packages/store/src/workspace/workspace.ts

```diff
@@ -444,7 +444,8 @@ export class Workspace {
     });
   }
 
-  createPage(pageId: string, parentId?: string) {
+  createPage(parentId?: string) {
```

Comment:
I think we can support `createPage(id? string)` that makes it possible to override the generated id (so that we can construct the `page0` without breaking ids), and having a new API for setting `parentId`.

Reply:
I plan to modify the API to the following form:

`createPage(options: { id?: sting, parentId?: string } = {})`

Reply:
LGTM since we have `new Workspace({ id })` API